inputPage.php

```diff
@@ -0,0 +1,62 @@
+<?php
+require_once('functions.php');
+$db = linkDB();
+$mice = getDataFromDB($db);
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <?php
+    require_once('HTMLhead.php');
+    ?>
+</head>
+
+<body>
+<nav>
+    <?php
+    require_once('HTMLnav.php');
+    ?>
+</nav>
+<main class="inputPageMain">
+    <h2>Add to the Mouse collection!</h2>
+    <div class = "inputPageDiv">
+        <form class="inputPageForm">
+            <label>
```

Comment:
I think you specify label/input pairings via for/id tags?

Reply:
can you elaborate what you mean here?